web/html/src/components/input/InputBase.js

```diff
@@ -85,6 +85,15 @@ export class InputBase extends React.Component<Props, State> {
     }
   }
 
+  componentDidUpdate(prevProps) {
+    // Support validation when changing the following props on-the-fly
+    if (this.props.required !== prevProps.required ||
+        this.props.disabled !== prevProps.disabled ||
+        this.props.validators !== prevProps.validators) {
```

Comment:
Do we actually need that the validators comparison? This one is always true even when it shouldn't and leads to infinite loops in the VM creation dialog at least. Even using lodash `isEqual` didn't fix it here.